caprover/gc-stack-deploy/src/gc_stack_deploy/stack_deploy.py

```diff
@@ -421,7 +451,7 @@ def copy_example(dest):
     dest : Path
         Destination file to write
     """
-    examples = importlib.resources.files("stack_deploy.example_configs")
+    examples = importlib.resources.files("gc_stack_deploy.example_configs")
```

Comment:
Oh wow, ruff autoformatter strikes again! _This_ here is the only intended change. `--config-file` was not working due to incorrect path from the latest efforts to rename the package.